scanner/src/main/kotlin/scanners/scanoss/ScanOss.kt

```diff
@@ -120,16 +115,11 @@ class ScanOss internal constructor(
                 "The $scannerName server returned an UUID not present in the mapping."
             }
 
-            fileNamesAnonymizationMapping[uuid] to entry.value
+            fileNamesAnonymizationMapping[uuid]!! to entry.value
```

Comment:
Instead of introducing `!!`, how about (in a preceeding commit) replacing the code incl. `require()` above with

            val fileName = fileNamesAnonymizationMapping[uuid] ?: throw IllegalArgumentException(
                "The $scannerName server returned an UUID not present in the mapping."
            )

            fileName to entry.value